src/eth/rpc/rpc_server.rs

```diff
@@ -553,6 +569,11 @@ async fn stratus_get_subscriptions(_: Params<'_>, ctx: Arc<RpcContext>, ext: Ext
     Ok(response)
 }
 
+/// Returns the count of executed transactions waiting to enter the next block.
+fn stratus_pending_transactions_count(_: Params<'_>, ctx: &RpcContext, _: &Extensions) -> usize {
+    ctx.storage.pending_transactions().len()
+}
+
```

Comment:
Does it make sense to expose this information in the state endpoint? Instead of creating a new endpoint